{
    "tests/test_data_client.py::test_status_handling[301-HacsException]": {
        "https://api.github.com/repos/hacs/integration": 1,
        "https://api.github.com/repos/hacs/integration/contents/custom_components/hacs/manifest.json": 1,
        "https://api.github.com/repos/hacs/integration/contents/hacs.json": 1,
        "https://api.github.com/repos/hacs/integration/git/trees/main": 1,
        "https://api.github.com/repos/hacs/integration/releases": 1,
        "https://data-v2.hacs.xyz/integration/repositories.json": 1
    }
}